core/src/cg/group4/game_logic/stroll/events/multiplayer_event/Host.java

```diff
@@ -1,9 +1,20 @@
 package cg.group4.game_logic.stroll.events.multiplayer_event;
```

Comment:
I'm not sure whether its a smart idea to push these changes to Host to the `dev` is a good idea. 
This code will not merge with the `dev` (and will fail the tests).

I assume you pulled in another branch from which you inherited this. 
It should already be fixed on the `dev` branch though.